docroot/profiles/humsci/su_humsci_profile/su_humsci_profile.info.yml

```diff
@@ -27,7 +27,6 @@ dependencies:
   - ckeditor
   - color_field
   - components
-  - config_readonly
```

Comment:
:( no more dependency?

Reply:
Oh I see why.

Reply:
during install it causes issues. once we get the hs_config_readonly i plan to add that as a dependency after installation. https://github.com/SU-HSDO/suhumsci/blob/test-multisite/docroot/profiles/humsci/su_humsci_profile/su_humsci_profile.install#L57